include/experimental/__p1673_bits/blas3_matrix_product.hpp

```diff
@@ -1206,23 +1170,26 @@ void hermitian_matrix_left_product(
   std::experimental::mdspan<ElementType_C, std::experimental::extents<numRows_C, numCols_C>, Layout_C, Accessor_C> C)
 {
   using size_type = typename extents<>::size_type;
+  using std::conj;
 
   if constexpr (std::is_same_v<Triangle, lower_triangle_t>) {
     for (size_type j = 0; j < C.extent(1); ++j) {
-      for (size_type i = j; i < C.extent(0); ++i) {
+      for (size_type i = 0; i < C.extent(0); ++i) {
         C(i,j) = ElementType_C{};
         for (size_type k = 0; k < A.extent(1); ++k) {
-          C(i,j) += A(i,k) * B(k,j);
+          ElementType_A aik = i <= k ? conj(A(k,i)) : A(i,k);
```

Comment:
I don't think this will compile for generic `ElementType_A`, because `std::conj` always returns `std::complex` (even if its argument is real).  There might be something in here like a "conditional `conj`" that returns the same type as its argument, but it's not too hard to write one:

```cpp
template<class T>
T cond_conj(const T& t) {
  return t;
}
template<class R>
std::complex<R> cond_conj(const std::complex<R>& z)
{
  using std::conj;
  return conj(z); 
}
```